reference/docs-conceptual/learn/security-features.md

```diff
@@ -89,6 +89,20 @@ The table below outlines the features that meet the servicing criteria and those
 - PowerShell 7.2 now disallows scripts from using COM objects in AppLocker system lock down
   conditions. Cmdlet that use COM or DCOM internally are not affected.
 
+## Software Bill of Materials (SBOM)
+
+Beginning with PowerShell 7.2, all install packages contain a Software Bill of Materials (SBOM). The
+SBOM is found at `$PSHOME/_manifest/spdx_2.2/manifest.spdx.json`. The creation and publishing of the
+SBOM is the first step to modernize Federal Government cybersecurity and enhance software supply
+chain security.
+
+The PowerShell team is also producing SBOMs for modules that they own but ship separately from
+PowerShell. For modules, the SBOM is installed in the module's folder under
```

Comment:
```suggestion
New versions of modules, shipped separately PowerShell and published by PowerShell team, will also contain SBOMs.
For modules, the SBOM is installed in the module's folder under
```

I don't mean this to be exact language, just a clarification that only new versions will get the SBOM.